src/xava.c

```diff
@@ -51,24 +51,36 @@ static XAVA xava;
 static pthread_t p_thread;
 
 void handle_ionotify_call(XAVA_IONOTIFY_EVENT event, const char *filename,
-        int id, XAVA *xava) {
-    UNUSED(id);
-    UNUSED(xava);
+        int id, XAVA *xavaionot) {
     UNUSED(filename);
-    switch(event) {
-        case XAVA_IONOTIFY_CHANGED:
-        //case XAVA_IONOTIFY_DELETED: // ignore because it is just broken
-            should_reload = 1;
-            break;
-        case XAVA_IONOTIFY_ERROR:
-            xavaBail("ionotify event errored out! Bailing...");
+    UNUSED(xavaionot);
+    switch(id) {
+        case XAVA_IONOTIFY_CALLBACK_MAIN:
+            printf("triggered non pywal!\n");
```

Comment:
You should remove the debug printf's once you're done debugging ;)

Reply:
Also for future reference use xavaLog or xavaSpam for these kinds of purposes, it's arguments are the same as printf's.